File: 백준/Bronze/1546. 평균/평균.java
import java.util.Arrays;
import java.util.Scanner;
import java.util.stream.IntStream;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int howManyNum = Integer.parseInt(scanner.nextLine());
        String inputString = scanner.nextLine();

        int[] classScore = Arrays.stream(inputString.split(" ")).mapToInt(Integer::parseInt).toArray();
        int sumClassScore = Arrays.stream(classScore).sum();
        int maxClassScore = Arrays.stream(classScore).max().getAsInt();
        double result = (double) (sumClassScore * 100) / maxClassScore / howManyNum;
        System.out.println(result);
    }
}

File: 백준/Bronze/1546. 평균/평균.kt
import java.io.*

fun main() = with(BufferedReader(InputStreamReader(System.`in`))) {

    val countOfClass = readLine().toInt()
    val classScore = readLine().split(' ').map { it.toInt() }

    var max = 0
    var sum = 0.0
    classScore.forEach {
        max = kotlin.math.max(max, it)
        sum += it
    }

    val result = sum / max * 100 / countOfClass

    print(result)
}